src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,237 @@
+---
+title: CI/CD for API Mesh
+description: Learn how to set up continuous integration and continuous delivery/deployment for API Mesh .
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# CI/CD for API Mesh
+
+API Mesh for Adobe Developer App Builder now provides CI/CD through [Github actions](https://docs.github.com/en/actions). This allows developers to build, test, and deploy meshes faster and more reliably.
+
+## API Mesh authentication
```

Comment:
I've added some numbering. @prabhuram93  Could you clarify what you mean when comparing it to the App Builder docs?

Reply:
Numbering works. I liked the way how this has been split into Lessons. It made it easy to comprehend. It may not apply to this, so just a recommendation to consider the possibility.